hicexplorer/hicExport.py

```diff
@@ -197,12 +204,11 @@ def main(args=None):
             hic_ma.matrix = (triu(hic_ma.matrix, k=-limit) - triu(hic_ma.matrix, k=limit)).tocsr()
             hic_ma.matrix.eliminate_zeros()
 
-    if not args.inputFormat == 'cool' and args.chromosomeOrder is not None and len(args.chromosomeOrder) == 1:
-        if args.chromosomeOrder:
-            hic_ma.keepOnlyTheseChr(args.chromosomeOrder)
+    if args.chromosomeOrder and are_chrom_reordered is False:
```

Comment:
I opted for adding the flag `are_chrom_reordered` to fulfill the python manifesto: "Explicit is better than implicit" and "Readability counts"